config/tutorials/tutorials.json

```diff
@@ -41,25 +41,19 @@
     "sdk": {
       "id": "sdk",
       "name": "Using Ably Client Library SDKs",
-      "desc": "A selection of tutorials demonstrating the use of Realtime and/or REST libraries within devices such as browsers, desktops, mobiles or servers."
-    },
-    "kafka": {
-      "id": "kafka",
-      "tab": "Channels, Push, Reactor & API Hub",
-      "name": "With Using Ably with Kafka",
-      "desc": "Tutorials on using Kafka with Ably."
+      "desc": "A selection of tutorials demonstrating the use of the Ably Client library SDKs, or the Ably REST API, within devices such as browsers, desktops, mobiles or servers."
```

Comment:
I think I'd be inclined to say "on devices such as" rather than "within devices"?

Reply:
Agreed. There's a few things I'd reword on that page! I've fixed a couple while I was there.